src/main/java/core/basesyntax/FigureInterface.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface FigureInterface {
```

Comment:
This is a second example of a bad naming)
Figureinterface does not give us a lot of information about the behavior of its implementations. Let's fix it